src/web/app/components/question-responses/grq-rgq-view-responses/grq-rgq-view-responses.component.ts

```diff
@@ -68,31 +69,35 @@ export class GrqRgqViewResponsesComponent extends ResponsesInstructorCommentsBas
   private filterResponses(): void {
     this.responsesToShow = {};
     this.teamsToUsers = {};
+    this.usersToTeams = {};
     this.teamExpanded = {};
     this.userExpanded = {};
     for (const question of this.responses) {
       for (const response of question.allResponses) {
         if (this.isGrq) {
           this.teamsToUsers[response.giverTeam] = this.teamsToUsers[response.giverTeam] || [];
+          this.usersToTeams[response.giver] = this.usersToTeams[response.giver] || [];
           if (this.teamsToUsers[response.giverTeam].indexOf(response.giver) === -1) {
             this.teamsToUsers[response.giverTeam].push(response.giver);
+            this.usersToTeams[response.giver].push(`(${response.giverTeam})`);
```

Comment:
Till this point, you are still dealing with **data**. So let's not do data formatting here (i.e. remove the brackets)